lib/Reports/AllUsers.php

```diff
@@ -80,13 +80,15 @@ public function printReport(InputInterface $input, OutputInterface $output) {
 			'quota' => $this->config->getAppValue('files', 'default_quota', FileInfo::SPACE_UNKNOWN),
 			'shares' => 0,
 			'login' => 0,
+			'display_name' => '',
 		];
 
 		$progress = new ProgressBar($output);
 
 		$i = 0;
 		$this->userManager->callForAllUsers(function(IUser $user) use ($default, $progress, &$i) {
 			$this->reports[$user->getUID()] = $default;
+			$this->reports[$user->getUID()]['display_name'] = $user->getDisplayName();
```

Comment:
can you put this behind a config, similar to `if ($input->getOption('last-login')) {`
Otherwise this will slow down the app very much for installations with millions of users